assets/js/util/i18n.js

```diff
@@ -285,8 +289,43 @@ export const numFmt = ( number, options = {} ) => {
  */
 export const numberFormat = ( number, options = {} ) => {
 	const { locale = getLocale(), ...formatOptions } = options;
+	try {
+		/**
+		 * Per https://github.com/google/site-kit-wp/issues/3255 there have been issues with some versions of Safari
+		 * on some operating systems throwing issues with some parameters in the formatOptions.
+		 *
+		 * If an error is thrown, we remove some troublesome params from the formatOptions object and fallback to no formatting.
+		 *
+		 * This allows us to degrade somewhat gracefully without breaking the dashboard for users of unaffected browsers.
+		 */
+		return new Intl.NumberFormat( locale, formatOptions ).format( number );
+	} catch ( error ) {
+		logOnce( `Unable to: new Intl.NumberFormat( ${ JSON.stringify( locale ) }, ${ JSON.stringify( formatOptions ) } ).format( ${ number } );`, 'warn' );
+
+		// Remove these key/values from formatOptions and try again.
+		const removeFormatOptions = {
+			currencyDisplay: 'narrow',
+			currencySign: 'accounting',
+			style: 'unit',
+			signDisplay: '*', // * denotes any value.
+			compactDisplay: '*',
+		};
 
-	return new Intl.NumberFormat( locale, formatOptions ).format( number );
+		const reducedFormatOptions = {};
+
+		for ( const [ key, value ] of Object.entries( formatOptions ) ) {
+			if ( removeFormatOptions[ key ] && [ value, '*' ].includes( removeFormatOptions[ key ] ) ) {
```

Comment:
The second part of the conditional here is a little clever 😄 

Given the comments above, let's make this a bit more straightforward using separate variables so we can explicitly check by key and value, or just the key.

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3620/commits/96d0593c48ce49b0227e787020007e9e966f7a7e